### PR TITLE
Feat: Refine shortest API

### PR DESCRIPTION
**Pull request by m2rads**
v0.0.4 

Changes: 
- Simplify shortest API
- Improve Test Reporting 
- Improve Navigation 
- Simplify shortest.config.ts file
- Remove browser session persistence 
- Add support for playwrights page object 
- Move screenshot folder to .shortest and add periodic cleanup
- Update docs

dashboard.test.ts
```typescript
import { test } from '@antiwork/shortest';

const githubCredentials = {
    username: process.env.GITHUB_USERNAME,
    password: process.env.GITHUB_PASSWORD,
}
  
test('Login to the app using Github login', githubCredentials)
test('clicking write new test button should initiate test generation')
```

shortest.config.ts
```typescript
import type { ShortestConfig } from '@antiwork/shortest';

export default {
  headless: false,
  baseUrl: 'http://localhost:3000',
  testDir: 'app/__tests__',
  anthropicKey: process.env.ANTHROPIC_API_KEY
} satisfies ShortestConfig; 
```
 

https://github.com/user-attachments/assets/751eb284-b7d8-4ad2-83f6-c9b3779c5f26

### CODE REVIEW

**Comment by m2rads** [comment]
Hi @slavingia, once the package is published I will test it in Iffy to make sure it installs properly and then record the demo.

File: README.md [hunk 1]
```diff
@@ -1,44 +1,85 @@
...
+
+```typescript
+import { test } from '@antiwork/shortest'
+
+const loginCredentials = {
+  email: process.env.EMAIL,
+  password: process.env.PASSWORD,
+}
 
+test('Login to the app using email and password', githubCredentials)
```
Comment (slavingia) [inline README.md:46]: Can we just use the dictionary here? No real point in creating it imo, and adds lines to the example.

Comment (m2rads) [reply README.md:46]: Well actually we can just have plain email and password in the test description. It's cleaner. Should I do that?

Comment (slavingia) [reply README.md]: I'm open to supporting both, I agree that inline is probably easier/simpler though.

Comment (m2rads) [reply README.md]: Sure I added the dictionary way btw. Let me know if there is something else.

**Comment by m2rads** [comment]
Hi @slavingia I addressed the issue #105 in this PR. Please let me know if you'd like to refine something else.

**Review by slavingia — APPROVED** [review]

**Pull request merged**